dbms/src/Formats/BlockInputStreamFromRowInputStream.h

```diff
@@ -22,6 +22,7 @@ class BlockInputStreamFromRowInputStream : public IBlockInputStream
         const RowInputStreamPtr & row_input_,
         const Block & sample_,
         UInt64 max_block_size_,
+        UInt64 min_block_size_,
```

Comment:
Name is not good. It's how often we check limits, and there is no connection with block size.